runtime/tests/third_party/dummy_device.toml

```diff
@@ -1,123 +1,112 @@
-schema = 1
+schema = 2
 
-[device]
-name = "dummy.device.qubit"
+[operators.gates.native]
 
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "Projector",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
-
-# The union of all gate types listed in this section must match what
-# the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "QubitUnitary",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "MultiRZ",
-        "Hadamard",
-        "S",
-        "T",
-        "CNOT",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "CY",
-        "CZ",
-        "PhaseShift",
-        "ControlledPhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "Identity",
-        "IsingXX",
-        "IsingYY",
-        "IsingZZ",
-        "IsingXY",
-]
+QubitUnitary = { properties = [ "controllable", "invertible"]  }
```

Comment:
This is just the dummy device, since you said you wanted it to mirror lightning remember that the gates in lightning also need the `differentiable` property specified (not all gates).

Reply:
Oh, you are right. But this is weird: we don't check the differentiable flag in any way besides syntactically now. I will do my best to set it correctly, but the risk of mistake would be high.

Reply:
I also found this list which I believe is also related to differentiation support:
https://github.com/PennyLaneAI/pennylane-lightning/blob/bb9b1443efd6e3f4e6be0865303f92b4a1c733f9/pennylane_lightning/core/src/gates/GateOperation.hpp#L109

But the Python requirements you listed above might be sufficient, you might just want to double check with someone from perf.

Reply:
But I'm also happy to merge as is and only add the differentiable property in the PR to the lightning repo.

Reply:
Following the logic which requires us to treat left-hand sides of gate config assignments as names of PL classes, we probably should rely on the criteria I listed above, rather than on device internal gate identifiers..

Reply:
Agreed TOML should contain PL class names, but the names should be the same anyways (because we pass the PL class names down to lightning as gate identifiers). I just mean in terms of what gate supports what.

Reply:
I checked the differentiability - it seems that all the gates that we list are differentiable, except the CRot. I'm checking it with the performance team